package.json

```diff
@@ -33,7 +33,7 @@
     "chalk": "^5.0.0",
     "cosmiconfig": "^9.0.0",
     "decamelize": "^6.0.0",
-    "flat-cache": "^5.0.0",
+    "flat-cache": "^6.1.4",
```

Comment:
This has to be specifically 6.1.4 or higher because 6.1.4 is the first release that implements a fix for https://github.com/jaredwray/cacheable/issues/926